projects/yumiswap/index.js

```diff
@@ -0,0 +1,37 @@
+const { calculateUsdUniTvl } = require('../helper/getUsdUniTvl')
+const { stakingUnknownPricedLP } = require("../helper/staking");
+
+const FACTORIES = "0xD3CFB8A232Ad5D0A7ABc817ae3BD1F3E7AE4b5E0"
+const YUMI_TOKEN_ADDRESS = "0x665F8B21878bDECa8bD94507120730a40dCd4F61";
+const XYUMI_ADDRESS = "0xCc7a1Bd54375fa5bc408aC319569eCc81c8d4208";
+
+const NATIVE_TOKEN_WASTR = "0xAeaaf0e2c81Af264101B9129C00F4440cCF0F720"
+
+const TOKENS = {
+    WASTR: NATIVE_TOKEN_WASTR,
+    WETH: "0x81ECac0D6Be0550A00FF064a4f9dd2400585FE9c",
+    WBTC: "0xad543f18cFf85c77E140E3E5E3c3392f6Ba9d5CA",
+    WBNB: "0x7f27352D5F83Db87a5A3E00f4B07Cc2138D8ee52",
+    MATIC: "0xdd90E5E87A2081Dcf0391920868eBc2FFB81a1aF",
```

Comment:
the 4th param of calculateUsdUniTvl() should only include high liquidity tokens. This is because these whitelist tokens are used as price oracles for all the pairs we iterate through. Could you remove some of the low liquidity tokens (as long as they dont have a big negative impact on TVL)? This is because low liquidity pools can cause huge swings in TVL using this function

Reply:
Hi @waynebruce0x
Now TVL is low as farming is not live yet.
we provide YUMI farming on the current tokens pair.
You can check the farms here
https://yumiswap.com/farms
It will be increased as farming goes live on 5th May.
We are gonna build https://github.com/DefiLlama/DefiLlama-Adapters/pull/1 dex on Astar.
Can we keep the current tokens in TVL calculation?
Thanks.

Reply:
no because for example MATIC only has $7 of liquidity so its not a reliable oracle. The MATIC pool will still be counted without MATIC being on the whitelist. Please remove YUMI, MATIC, BUSD and BNB

Reply:
got it
I removed.
please check
thanks